internal/commands/login/inputs_test.go

```diff
@@ -16,6 +17,7 @@ func TestLoginInputs(t *testing.T) {
 		prepareProfile func(p *user.Profile)
 		procedure      func(c *expect.Console)
 		test           func(t *testing.T, i inputs)
+		expectBrowser  bool
```

Comment:
[meganit] seems like you could have saved yourself some key strokes if you flipped this to `expectNoBrowser bool` 😬